src/Nethermind/Nethermind.Blockchain/Comparers/GasPriceTxComparer.cs

```diff
@@ -35,6 +34,7 @@ public GasPriceTxComparer(IBlockTree blockTree, ISpecProvider specProvider)
         
         public int Compare(Transaction? x, Transaction? y)
         {
+            // When we don't know next block base fee, we're getting it from the current Head. That should be great approximation
```

Comment:
I would like to learn more about it?

Reply:
we can calculate it from head, right?

Reply:
Yes. I changed that comment a bit.
That is how I see it:
When we're adding Tx to TxPool we don't know the base fee of the block in which transaction will be added. What we can do is get a base fee from the head in two way:
1. head.Header.BaseFee - the current one
2. BlockHeader.CalculateBaseFee(head, spec) - but I am not sure if it is worth recalculating a base fee here